src/components/MoneyRequestHeader.js

```diff
@@ -90,6 +94,22 @@ const MoneyRequestHeader = (props) => {
     const shouldShowSettlementButton = !isSettled && !props.isSingleTransactionView && isPayer;
     const bankAccountRoute = ReportUtils.getBankAccountRoute(props.chatReport);
     const shouldShowPaypal = Boolean(lodashGet(props.personalDetails, [moneyRequestReport.managerEmail, 'payPalMeAddress']));
+
+    const deleteTransaction = useCallback(() => {
+        IOU.deleteMoneyRequest(parentReportAction.originalMessage.IOUTransactionID, parentReportAction, true);
+    }, [parentReportAction]);
+
+    if (props.isSingleTransactionView && isDeletedParentAction) {
+        return (
+            <HeaderView
+                reportID={props.report.reportID}
+                onNavigationMenuButtonClicked={() => Navigation.goBack(ROUTES.HOME)}
+                personalDetails={props.personalDetails}
+                report={props.report}
```

Comment:
Why do we need to pass both, report and reportID?

Reply:
Good point. This was copy/pasta from ReportScreen. I removed it in both. Thanks!